docs/cudf/source/user_guide/pandas-comparison.md

```diff
@@ -107,10 +114,11 @@ Compare the results obtained from Pandas and cuDF below:
  10  640.00
 ```
 
-To match Pandas behavior, you must explicitly pass `sort=True`:
+If you require that results be returned in a predictable order, you
+must pass the `sort=True` option explicitly:
```

Comment:
Is it worth mentioning about the index being useful to write ordering-independent code?

Reply:
Done!